c++/src/kj/string.h

```diff
@@ -734,6 +740,9 @@ inline StringPtr StringPtr::slice(size_t start) const {
 inline ArrayPtr<const char> StringPtr::slice(size_t start, size_t end) const {
   return content.slice(start, end);
 }
+inline ArrayPtr<const char> StringPtr::first(size_t count) const {
+  return content.slice(0, count);
```

Comment:
This could be `content.first(count);`, right?

Reply:
I actually prefer it to be `this.slice(0, count)` everywhere. fixed this place too.